### PR TITLE
[Mailer] add new events

### LINKED ISSUE #17071: [Mailer] Add new events
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#47080
| PR author(s) | @fabpot
| Merged in    | 6.2

### PR DESCRIPTION
**Pull request by mdoutreluingne**
Fixes #17071

### CODE REVIEW

File: mailer.rst [hunk 1]
```diff
@@ -1437,6 +1437,65 @@ is sent::
...
+            // e.g you can get more information on this error when sending an email.
+            $event->getError();
+
+            // do something with the message
+        }
+    }
+
+.. versionadded:: 6.2
+
+    ``SentMessageEvent`` and ``FailedMessageEvent`` was introduced in Symfony 6.2.
```
Comment (94noni) [inline mailer.rst]: were introduced

File: mailer.rst [hunk 2]
```diff
@@ -1437,6 +1437,65 @@ is sent::
         }
     }
 
+SentMessageEvent
+~~~~~~~~~~~~
+
+``SentMessageEvent`` it allows acting on the :class:`Symfony\\Component\\Mailer\\SentMessage`::
```
Comment (94noni) [inline mailer.rst]: Perhaps document like this?
https://symfony.com/doc/current/reference/events.html#kernel-request

When and why and what can be done in it

File: mailer.rst (hunk 1 above)
Comment (mdoutreluingne) [reply mailer.rst]: done

File: mailer.rst (hunk 2 above)
Comment (mdoutreluingne) [reply mailer.rst]: Good idea !
Personally I like it when it's uniform (it's simpler and nicer to read). So I standardized like https://symfony.com/doc/current/reference/events.html#kernel-request 

I have detailed this event. Thanks for your review

**Comment by javiereguiluz** [comment]
Nice contribution! Thanks Maxime ... and thanks to reviewers too!

**Pull request merged**